dsgrid/config/dimension_config.py

```diff
@@ -140,7 +130,10 @@ def get_tzinfo(self):
 
 
 class AnnualTimeDimensionConfig(TimeDimensionBaseConfig):
-    """Provides an interface to an AnnualTimeDimensionModel."""
+    """
+    Provides an interface to an AnnualTimeDimensionModel.
+    AnnualTimeDimension is primarily for constant annual growth factors
```

Comment:
There are going to be plenty of data at the annual level, not just growth factors.